packages/lab/tsconfig.json

```diff
@@ -5,9 +5,7 @@
     "baseUrl": "src",
     "rootDir": "src",
     "outDir": "dist",
-    "paths": {
```

Comment:
We need to keep this alias configuration. Otherwise we'll get a bunch of errors when building the package 🙏